Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -685,7 +685,7 @@ def bottle_formula(formula)
               "root_url" => bottle.root_url,
               "cellar"   => bottle_cellar.to_s,
               "rebuild"  => bottle.rebuild,
-              "date"     => Pathname(filename.to_s).mtime.strftime("%F"),
+              "date"     => Pathname(filename.to_s).mtime.iso8601,
```

Comment:
I think Ruby outputs RFC3339 compliant date/time with `.iso8601`? At least `DateTime` library mentions both are equivalent to `strftime('%FT%T%:z')`.

Renaming to `datetime` would be more accurate if preferred.

Reply:
Added comment. Also added `utc` for more consistent formats.

I guess some other alternatives would be to keep the `date` and just go without time like `#{date}T00:00:00Z` or to save a separate key for `time`/`datetime`. Just checking if anything else is preferred.